ui-guidelines/css/resources.css

```diff
@@ -0,0 +1,128 @@
+/*==== TYPOGRAPHY ====*/
+@import url(https://fonts.googleapis.com/css?family=Open+Sans);
+
+.font-weights {
+	width: 100%;
+	height: 10em;
+}
+.weight-sample {
+	width: 20%;
+	float: left;
+	margin-right: 5%;
+	border: 1px solid #fce3da;
+	border-radius: 8px;
+}
+.big-letter{
+	font-size: 6em;
+	font-family: 'Open Sans', Arial, sans-serif;
+}
+.light {font-weight: 300;}
+.normal {font-weight: 500;}
+.medium {font-weight: 600;}
+.bold {font-weight: 700;}
+.text-size {margin-left: 1.5em;}
+
+.typography, .typography p {
```

Comment:
Anyway, when you set the `font-family` on `.typography` then you can remove the `.typography p` selector as it inherits `font-family` property from its parent.